tests/integration_tests/test_debug.py

```diff
@@ -133,6 +134,19 @@ def test_single_sensor_packet(qtbot, single_connection_tantalus):
                                                     DataEntryIds.TIME) == 0xFFFFFFFF
         assert app.rocket_data.last_value_by_device(DeviceType.TANTALUS_STAGE_1_FLARE, data_entry_id) == val
 
+def test_event_packet(qtbot, single_connection_tantalus):
+    app = single_connection_tantalus
+    connection = app.connections['DEBUG_CONNECTION']
+    packet = radio_packets.event(0xFFFFFFFF, 0x00)
+    snapshot = get_event_stats_snapshot()
+    connection.receive(packet)
+
+    assert EVENT_EVENT.wait(snapshot) == 1
+    assert app.rocket_data.last_value_by_device(DeviceType.TANTALUS_STAGE_1_FLARE,
+                                                DataEntryIds.TIME) == 0xFFFFFFFF
+    assert app.rocket_data.last_value_by_device(DeviceType.TANTALUS_STAGE_1_FLARE,
+                                                DataEntryIds.EVENT) == DataEntryValues.EVENT_ARMED.value
```

Comment:
As per the other comment.
```suggestion
                                                DataEntryIds.EVENT) == DataEntryValues.EVENT_ARMED
```